### DISCUSSION TITLE
The display problem of ttkbootstrap

### DISCUSSION THREAD

**Discussion by jiangwu007** [israel-dryer/ttkbootstrap#341] [Q&A]
When I set the window to 300 wide and 150 high, it will display normally. But if I change to another computer, the window display will be abnormal. Whether ttkbootstrap can automatically switch to normal display. I use Windows computer
![display1](https://user-images.githubusercontent.com/21976698/188532585-dcd5d805-8684-4afc-be69-0a4dea32b444.png)
![display](https://user-images.githubusercontent.com/21976698/188532614-8d6c3e87-b582-4bb0-842b-0d96a87f20dd.png)
Tkinter does not have this problem. Below, I set the width to 220 and the height to 110.
![tkinter1](https://user-images.githubusercontent.com/21976698/188550536-7b054c84-2cc3-45d9-804a-45134bc793f2.png)
![tkinter2](https://user-images.githubusercontent.com/21976698/188550090-15b09548-a8de-42f7-b0b4-494fad484ce1.png)
Can ttkbootstrap be designed like tkiner and automatically adjusted to the appropriate size of the form？

**Comment by israel-dryer** [comment]
Possibly this is related to enabling high DPI awareness on Windows. You'll notice the tkinter windows are a bit blurry compared to the ttkbootstrap windows because of the difference in how high dpi is enabled. However, this also affects how the window is scaled.

I recommend not setting a fixed window size and instead relying on the geometry manager to update the size for you. You could also try the `minsize` and `maxsize` parameters to set boundaries for your window size.

Finally, you may be able to get the result you want by disabling high DPI support:
```python
root = tk.Window(hdpi=False, size=(300, 150))  # you'll need to update size based on non-hdpi settings
```

**Reply by jiangwu007** [reply]
> Possibly this is related to enabling high DPI awareness on Windows. You'll notice the tkinter windows are a bit blurry compared to the ttkbootstrap windows because of the difference in how high dpi is enabled. However, this also affects how the window is scaled.
> 
> I recommend not setting a fixed window size and instead relying on the geometry manager to update the size for you. You could also try the `minsize` and `maxsize` parameters to set boundaries for your window size.
> 
> Finally, you may be able to get the result you want by disabling high DPI support:
> 
> ```python
> root = tk.Window(hdpi=False, size=(300, 150))  # you'll need to update size based on non-hdpi settings
> ```

Thank you very much for your help I wrote some exercises with ttkbootstrap. Maybe I can add some cases in ttkbootstrap gallery to show that ttkbootstrap can make great effects. Finally, I hope that the ttkbootstrap community will become better and better.
https://github.com/jiangwu007/ttk_game
